packages/arg-parser/src/arg-mapper.ts

```diff
@@ -105,9 +105,7 @@ const MAPPINGS: {
   awsSecretAccessKey: (i, v) => setAWSKMS(i, 'secretAccessKey', v),
   awsSessionToken: (i, v) => setAWSKMS(i, 'sessionToken', v),
   awsIamSessionToken: (i, v) => setAuthMechProp(i, 'AWS_SESSION_TOKEN', v),
-  // @ts-expect-error AutoEncryption options have been renamed, drop the (outer) csflePath call after the next release
```

Comment:
Don't know what these comments meant (drop what where?), but I removed the comments so it will compile.

Reply:
The things that should be dropped have already been removed, I forgot to update the comment – all good here :)